etc/waterfox.profile

```diff
@@ -78,7 +79,8 @@ seccomp
 shell none
 tracelog
 
-# private-bin waterfox,which,sh,dbus-launch,dbus-send,env
+# waterfox requires a shell to launch on Arch. We can possibly remove sh though.
+# private-bin waterfox,which,sh,dbus-launch,dbus-send,env,sh,bash
```

Comment:
I know its commented, but could you remove the duplicated sh and add dash?

Reply:
@SpotComms Ok, fixed.